### PR TITLE
Add <map> example

### LINKED ISSUE #761: [HTML Examples] Image and multimedia ~ map
Need an HTML example for the `map` element
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/map

## Contributing

https://github.com/mdn/interactive-examples/blob/master/CONTRIBUTING-HTML.md

### PR DESCRIPTION
**Pull request by 2alin**
Fixes #761 

Another one full of fun :grin:.
I followed the guidelines discussed in #920. Hope I didn't make a mistake and everything is alright.

Because it's a demo for `<map>` and not `<area>` I decided to not saturate it with all different kinds of shapes and went only with `poly` (the `<area>` demo has all types implemented).

And again, the graphic was completely made by me from scratch in Inkscape, even the idea is original.  So, no problems of copyright :wink:.

### CODE REVIEW

File: live-examples/html-examples/image-and-multimedia/map.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+          href="https://developer.mozilla.org/docs/Web/HTML"
+          target="_blank" alt="HTML" />
+    <area shape="poly" coords="130,147,130,228,6,219,59,107"
+          href="https://developer.mozilla.org/docs/Web/CSS"
+          target="_blank" alt="CSS" />
+    <area shape="poly" coords="130,147,200,107,130,4,59,107"
+          href="https://developer.mozilla.org/docs/Web/JavaScript"
+          target="_blank" alt="JavaScript" />
+</map>
+<img usemap="#infographic" src="/media/examples/mdn-info2.png" alt="MDN infographic">
```
Comment (schalkneethling) [inline live-examples/html-examples/image-and-multimedia/map.html]: Nit: Code style: Explicitly close self closing elements.

Comment (2alin) [reply live-examples/html-examples/image-and-multimedia/map.html]: in ``?

Comment (schalkneethling) [reply live-examples/html-examples/image-and-multimedia/map.html]: Yup, so:

```
<img usemap="#infographic" src="/media/examples/mdn-info2.png" alt="MDN infographic" />
```

**Comment by 2alin** [comment]
Thank you for reviewing it and the fast merge :)

**Pull request merged**